tool/jt.rb

```diff
@@ -43,6 +43,22 @@
 METRICS_REPS = Integer(ENV['TRUFFLERUBY_METRICS_REPS'] || 10)
 DEFAULT_PROFILE_OPTIONS = %w[--cpusampler --cpusampler.Output=flamegraph]
 
+MRI_TEST_RETAG_RETAIN_PATTERNS = [
+  /hangs/i,
+  /slow/i,
+  /java\.lang\./,
+  /retain-on-retag/,
+  /OOM/,
+  /RubyVM/,
+  /ShouldNotReachHere/,
+  /LLVMLinkerException/,
+  /flaky/,
+  /spurious/,
+  /transient/,
+  /if RUBY_PLATFORM/,
+  /pid \d+ (exit|killed)/,
+]
```

Comment:
Yeah, possibly. We weren't accounting for that before my changes, either, but we can split off the reason string and just look at that.

Reply:
This is actually a bit trickier because the line matches on more than just the reason. We retain exclusions for platform guards as well. It's doable, just a bit ugly.